6.x/custom-validation-rules.md

```diff
@@ -0,0 +1,67 @@
+# Custom Validation Rules
+
+---
+
+<a name="about"></a>
+## About
+
+Along with the provided Laravel validation rules, we created a few custom validation rules that you can use to validate Backpack fields in an easier way.
+No more messing with `.*` and such on validation rules. Just define fluently what you want to validate.
+
+<a name="valid-upload-validation-rule"></a>
+## `ValidUpload` for `upload` field type
+
+The `ValidUpload` rule is used to validate the `upload` field type. Using the custom rule helps developer avoid to setting different validation rules for different operations.(create/update).
+
+```php
+// for the field
+CRUD::field('avatar')->type('upload');
+
+// you can write the validation rule as
+'avatar' => ValidUpload::fileRules(File::types(['jpg','png'])->max(2048))->attributeRules('required'),
```

Comment:
Wait... what's the API here? What is `fileRules()` and what is `attributeRules()`? There's a single file getting uploaded here soo... I don't get it...